shipyard_blueprints/smartsheets/shipyard_smartsheets/cli/download.py

```diff
@@ -0,0 +1,156 @@
+import argparse
+import os
+import smartsheet
+import sys
+import logging
+import requests
+import pandas as pd
+from shipyard_templates import ExitCodeException, Spreadsheets as ss
+from typing import Dict, List, Any
+
+# custom exit code
+EXIT_CODE_INVALID_SHEET_ID = 220
```

Comment:
Why here and not in the templates? May have to slightly adjust the name to user a more generic term like `EXIT_CODE_NOT_FOUND` `EXIT_CODE_INVALID_INPUT` etc

Reply:
I opted for this specific code for smartsheets because of a very specific error case. I'm not opposed to putting that to the templates as `EXIT_CODE_NOT_FOUND`

Reply:
I just thought of something with the more specific error codes: should we make this more unique coding just in case in a weird world where the templates has an exit code 220. perhaps for vendor specific error codes they are in the 100 range or another range we do not commonly use. Thoughts?